src/Pages/Detail/Review.js

```diff
@@ -1,188 +1,156 @@
+                                })}
+                            </ul>
+                            <div className="inputWrapper">
+                                <input placeholder="댓글을 등록해 주세요:)" 
+                                onChange={this.handleText}
+                                value={this.state.text}
+                                onKeyPress={this.pressEnter}/>
 
-                      <button
-                        onClick={this.addText}
-                        disabled={disabled ? false : true}
-                      >
-                        등록
-                      </button>
-                    </div>
-                  </div>
-                </div>
-                <div className="writerBox">
-                  <div className="writer">작성자</div>
-                  <div className="name">{this.props.user}</div>
+                                <button onClick={this.addText}
+                                disabled={disabled ? false : 
+                                true}>등록</button>
```

Comment:
```suggestion
                                disabled={!disabled}>등록</button>
```
그 자체로 불리언을 리턴하는 경우를 잘 활용해주세요